gnomad_qc/v4/resources/basics.py

```diff
@@ -147,7 +149,7 @@ def _remove_ukb_dup_by_index(
     # We don't need to do the UKB withdrawn and pharma remove list sample removal if
     # we're only keeping high quality or release samples since they will not be in
     # either of those sets.
-    if not high_quality_only and not release_only:
+    if not (high_quality_only or release_only or controls_only):
```

Comment:
I actually didn't realize Python allowed triple logical statements like this, v cool !